app/views/ping/index.text.erb

```diff
@@ -2,5 +2,6 @@
 Madeline
 App is running.
 Delayed Job is <%= @dj ? "" : "NOT " %>running.
+There are <%= @count %> jobs in the queue.
```

Comment:
Not necessary, but I was interested in the number for my testing. I thought it might be nice to have.